docs/user-guide.rst

```diff
@@ -296,6 +296,31 @@ Anitya provides several different versions schemes.
     recent scheme and the rest will be moved to bottom unsorted.
 
 
+Version Pattern
+===============
+
+The version pattern is used to extract and match version numbers from
+project releases. It allows for specifying the format of version numbers
+using regular expressions.
+
+Patterns -
+- ``YYYY``: Full year (e.g., 2006, 2016, 2106).
+- ``YY``: Short year (e.g., 6, 16, 106).
+- ``0Y``: Zero-padded year (e.g., 06, 16, 106).
+- ``MM``: Short month (e.g., 1, 2, ..., 11, 12).
+- ``0M``: Zero-padded month (e.g., 01, 02, ..., 11, 12).
+- ``WW``: Short week (since the start of the year) (e.g., 1, 2, 33, 52).
+- ``0W``: Zero-padded week (e.g., 01, 02, 33, 52).
+- ``DD``: Short day (e.g., 1, 2, ..., 30, 31).
+- ``0D``: Zero-padded day (e.g., 01, 02, ..., 30, 31).
+
+Notes:
```

Comment:
Use notes from restructured text https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes